### ISSUE TITLE
broken ldapadmin links in header

### ISSUE THREAD

**Issue by landryb** [georchestra/georchestra#1136]
A bit different from #1096 and fallout of #1010 i think - with this config:

```
cas/cas.properties:ldapadmin.contextpath=/ldapadmin/
header/header.properties:ldapadminPublicContextPath=/ldapadmin/
ldapadmin/js/config.js:    publicContextPath: "//ldapadmin/"
ldapadmin/ldapadmin.properties:publicContextPath=/ldapadmin/
```
In my header, the ldapadmin link looks like this:
```
<a href="${ldapadminPublicContextPath}/privateui/">
```

which resolves to http://georchestra.craig.fr/header/$%7BldapadminPublicContextPath%7D/privateui/

And the 'user account details' link looks like this:
```
<a href="${ldapadminPublicContextPath}/account/userdetails">
```
which resolves to
http://georchestra.craig.fr/header/$%7BldapadminPublicContextPath%7D/account/userdetails

I think this is due to the removal of the double // prefixing ldapadminPublicContextPath, and on top of this the value doesnt seem correctly read from the config.

My header index.jsp comes from the my own build template (ie georchestra/config/configurations/craig-template/header/WEB-INF/jsp/index.jsp, copied/adapted from georchestra/header/src/main/filtered-resources/WEB-INF/jsp/index.jsp) and is properly propagated/deployed to the webapp.

**Comment by landryb** [comment]
Maybe due to this commit bdbbab01600ce8d2b418adcf705431191c5540b7 ?

**Comment by landryb** [comment]
Hm probably not since this should only be for jetty...

**Comment by landryb** [comment]
As i expected reverting bdbbab0 locally doesnt fix the issue :) .. it's like the config file in `${georchestra.datadir}/header/header.properties` was ignored.

**Comment by landryb** [comment]
It's probably the case, because if i go to `view-source:http://georchestra.craig.fr/header/` -> default language, the fr value is not taken from the properties file.

**Comment by landryb** [comment]
Okay, added some debugging gunk to the jsp here https://github.com/georchestra/georchestra/blob/master/header/src/main/filtered-resources/WEB-INF/jsp/index.jsp#L30..

```
exception: No bean named 'class org.georchestra.commons.configuration.GeorchestraConfiguration' is defined
```
but there's the <%@page import pointing at it, and the class should be here...
```
unzip -l /data/tomcat/georchestra/webapps/header/WEB-INF/lib/georchestra-commons-15.12-SNAPSHOT.jar |grep Conf
     4783  2015-11-26 15:07   org/georchestra/commons/configuration/GeorchestraConfiguration.class

```

**Comment by landryb** [comment]
Don't ask me why, but this fixes the issue here. Whether some error message should be printed in case of exceptions like this in the jsp is beyond my knowledge :)

**Comment by landryb** [comment]
Fixed by #1138
